### ISSUE TITLE
Syntax error in script

### ISSUE THREAD

**Issue by JusakNowWorldLater** [GalAvineri/ISIC-Archive-Downloader#18]
Hello, I download all requirements and when I try to start script ( python download_archive.py / python3.6 download_archive.py) I get : 
  File "download_archive.py", line 93
    def download_descriptions(ids: list, descs_dir: str, num_processes: int) -> list:
                                 ^
SyntaxError: invalid syntax

Could you please help to find the cause?

**Comment by JusakNowWorldLater** [comment]
python3.6 download_archive.py 
Traceback (most recent call last):
  File "download_archive.py", line 1, in <module>
    from download_single_item import LesionImageDownloader as ImgDownloader, SegmentationDownloader as SegDownloader
  File "/home/vjusak/workspace/MyWorkCPP/SkinCancerDetector/NetSide/ISIC-Archive-Downloader/download_single_item.py", line 8, in <module>
    from PIL import Image
  File "/usr/lib/python3/dist-packages/PIL/Image.py", line 66, in <module>
    from PIL import _imaging as core
ImportError: cannot import name '_imaging'

**Comment by JusakNowWorldLater** [comment]
Hello, I have repaired.
Root cause of this issue : 
my *python* = 2.7.4 - this will not work, python3 was python3.5.4, when Installed python3.6 --  python3 was still python3.5.4 and pip3 didn't download Pillow and tqdm in path of python3.6. 
I used : 
{noformat}
sudo rm /usr/bin/python3
sudo ln -s /usr/bin/python3.6 /usr/bin/python3
{noformat}
 installed Pillow and tqdm again by pip3, enter in terminal : python3 download_archive.py 
And It works. Thanks for your scripts. :)
